### PR TITLE
Add sourceRepo to SystemInfo object

### PR DESCRIPTION
**Pull request by jcucsb**
Added string SourceRepo variable in systemInfo class and allowed it to be overriden in the application.properties file.

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -18,6 +18,7 @@ management.endpoints.web.exposure.include=mappings
 springfox.documentation.swagger.v2.path=/api/docs
 spring.jpa.hibernate.ddl-auto=update
 app.admin.emails=${ADMIN_EMAILS:${env.ADMIN_EMAILS:phtcon@ucsb.edu}}
+app.sourceRepo=${SOURCE_REPO:${SystemInfo.java.sourceRepo}}
```
Comment (pconrad) [inline src/main/resources/application.properties:21]: This should be something like the following:

```
app.sourceRepo=${SOURCE_REPO:${env.SOURCE_REPO:ucsb-cs156/proj-happycows}}
```

What you have, i.e. `SystemInfo.java.sourceRepo` doesn't have any meaning here.   You need to model it after what's on line 20.  

The meaning is: 
* use the value of SOURCE_REPO if present
* If not use the value of env.SOURCE_REPO if present
* If not, use the default value `ucsb-cs156/proj-happycows`

Then, by setting SOURCE_REPO in your `.env` file and your Heroku Config vars, you can override the default.

**Review by pconrad — CHANGES_REQUESTED** [review]
Needs one change, as explained below.

File: src/main/resources/application.properties (hunk 1 above)
Comment (heyyysus) [reply src/main/resources/application.properties]: Change has been implemented.

**Review by pconrad — CHANGES_REQUESTED** [review]
What you have is better!

But still one missing piece: You need to connect the dots in SystemInfoServiceImpl.java

That is, you need to get the value from `app.sourceRepo` and store it in the new data member that you created.  There are two examples in that file of how to do it.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -18,6 +18,7 @@ management.endpoints.web.exposure.include=mappings
 springfox.documentation.swagger.v2.path=/api/docs
 spring.jpa.hibernate.ddl-auto=update
 app.admin.emails=${ADMIN_EMAILS:${env.ADMIN_EMAILS:phtcon@ucsb.edu}}
+app.sourceRepo=${SOURCE_REPO:${env.SOURCE_REPO:ucsb-cs156/proj-happycows}}
```
[the remark this answers is not in the document]
Comment (jcucsb) [reply src/main/resources/application.properties]: Change has been implemented.

**Review by pconrad — APPROVED** [review]
LGTM

**Review by heyyysus — APPROVED** [review]
Passes tests and behaves correctly on localhost and production. LGTM.

**Pull request merged**